e2e/utils/controls.ts

```diff
@@ -183,3 +183,25 @@ export const clickOnText = async (text: string) => {
   await textElement.waitForEnabled({ timeout })
   await textElement.click()
 }
+
+export const swipeButton = async (title: string) => {
+  const sliderButton = await $(selector(title, "Other"))
+  await sliderButton.waitForDisplayed({ timeout })
+
+  const location = await sliderButton.getLocation()
+  const size = await sliderButton.getSize()
+
+  const thumbWidthPercentage = 0.1
+
+  const thumbWidth = size.width * thumbWidthPercentage
+  const startX = location.x + thumbWidth / 2
+  const startY = location.y + size.height / 2
+
+  const endX = location.x + size.width - thumbWidth
+  await driver.touchAction([
+    { action: "press", x: startX, y: startY },
+    { action: "wait", ms: 1000 },
+    { action: "moveTo", x: endX, y: startY },
+    "release",
```

Comment:
@Aditi-Singh16 
 we might just need to add another wait `{ action: "wait", ms: 5000 }` before `release` to make the e2e work better